code/game/objects/structures/crates_lockers/closets.dm

```diff
@@ -51,12 +60,20 @@
 	var/obj/effect/overlay/closet_door/door_obj
 	var/obj/effect/overlay/closet_door/door_obj_alt
 	var/is_animating_door = FALSE
-	var/door_underlay = FALSE //used if you want to have an overlay below the door. used for guncabinets.
-	var/door_anim_squish = 0.12 // Multiplier on proc/get_door_transform. basically, how far you want this to swing out. value of 1 means the length of the door is unchanged (and will swing out of the tile), 0 means it will just slide back and forth.
+
+	/// Used if you want to have an overlay below the door. used for guncabinets.
+	var/door_underlay = FALSE
+	/// Multiplier on proc/get_door_transform. basically, how far you want this to swing out. value of 1 means the length of the door is unchanged (and will swing out of the tile), 0 means it will just slide back and forth.
+	var/door_anim_squish = 0.12
 	var/door_anim_angle = 147
-	var/door_hinge = -6.5 // for closets, x away from the centre of the closet. typically good to add a 0.5 so it's centered on the edge of the closet.
-	var/door_hinge_alt = 6.5 // for closets with two doors. why a seperate var? because some closets may be weirdly shaped or something.
-	var/door_anim_time = 2.5 // set to 0 to make the door not animate at all
+	/// For closets, x away from the centre of the closet. typically good to add a 0.5 so it's centered on the edge of the closet.
+	var/door_hinge = -6.5
+	/// For closets with two doors. why a seperate var? because some closets may be weirdly shaped or something.
+	var/door_hinge_alt = 6.5
+	/// Set to 0 to make the door not animate at all
+	var/door_anim_time = 2.5
+	/// Set to TRUE to have the locker not be filled with items, useful for ruins where we dont want people to have the items inside
+	var/no_fill = FALSE
```

Comment:
no

the most ideal way to do it, is either `/empty` subtypes for empty lockers,

or even better, the main locker type is empty and with a `/whatever` subtype that does have items